docs/api.md

```diff
@@ -1,3 +1,13 @@
+<p align="right">
+<b>Powered by:</b>
```

Comment:
This "Powered by" looks pretty cheesy to me, and a bit out of place, spacing-wise. Can we remove that and just use the image?

![image](https://user-images.githubusercontent.com/8457307/41286532-db7b235c-6e0d-11e8-8539-7a4cd78e39b8.png)

If we want more, maybe drop a new heading at the bottom of the Index page, Maintained By, or Provided By, or somesuch, with a short paragraph about plus3it and some links to p3 resources (github, home page, etc).

Reply:
But I'm fine with just the image for now, and we can let that stew before doing anything more.

Reply:
@Briankwatson was fine with removing the text.  Code has been updated to reflect the change